### DISCUSSION TITLE
Error 500 for /api/v1/instance

### DISCUSSION THREAD

**Discussion by bilal-hanif43** [mastodon/mastodon#24324] [General]
I am encountering a 500 error on page refresh for request /api/v1/instance. 
I have others API calls that are working fine like 
/api/v1/markers
/api/v1/accounts/relationships?id[]=108759762874551565&id[]=108815608754207177
/api/v1/trends/tags

Please guide how to fix.

**Comment by mjankowski** [comment]
Did this ever resolve itself?

Can you add more logs?
